css/types/counter.json

```
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counter",
+          "description": "<code>counter()</code>",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
```

I'm going to guess that it's safe to go with the first release here

```suggestion
              "version_added": "18"
```

---

Correct.